libarchive/archive_read_append_filter.c

```diff
@@ -111,7 +111,7 @@ archive_read_append_filter(struct archive *_a, int code)
     number_bidders = sizeof(a->bidders) / sizeof(a->bidders[0]);
 
     bidder = a->bidders;
-    for (i = 0; i < number_bidders; i++, bidder++)
+    for (i = 1; i < number_bidders; i++, bidder++)
```

Comment:
I think this is correct, but the logic in this function is very strange.  Let's take this as-is for now, but it would be nice for someone to try to clean up and simplify this function.